### ISSUE TITLE
Engine validation for newly created action + validation for `capact action create`

### ISSUE THREAD

**Issue by mszostok** [capactio/capact#425]
### Engine [3SP]

- Validate user input for newly created Action
   - TypeInstance `typeRef` matches the one from input - we assume that the TypeInstance content is valid (as it will be validated in Hub)
   - Parameters against JSON schema from Interface [blocked by #399 ]
   - `additionalInput.typeInstances` typeRef + `additionalInput.parameters` against JSONSchema on Implementation
- Out of scope for now: nested validation of input and additional input

Before implementation, consider two options:
1. validate Action before rendering logic
2. validate Action during rendering

Compare two options and discuss them with team

#### `capact action create` for new Action creation
1. `Type.spec.jsonSchema` + `Interface.input.parameters`: validate if its content is valid JSON schema [1SP]
